apps/openmw/mwworld/containerstore.cpp

```diff
@@ -894,8 +897,17 @@ void MWWorld::ContainerStore::readState (const ESM::InventoryState& inventory)
         }
     }
 
-
-    mLevelledItemMap = inventory.mLevelledItemMap;
+    // Fix old saves
+    for(const auto& entry : inventory.mLevelledItemMap)
```

Comment:
We usually implement such things on components level (for example, in the `ESM::InventoryState::load`). In this case you can check ESM version, so you can skip such fallback iterations for new saves.